.github/scripts/windows/test_task.bat

```diff
@@ -131,6 +131,10 @@ for %%i in (ldap) do (
 	del %PHP_BUILD_DIR%\php_%%i.dll
 )
 
+rem reduce excessive stack reserve for testing
+editbin /stack:8388608 %PHP_BUILD_DIR%\php.exe
```

Comment:
just curious is cl.exe able to do it at compile time ?

Reply:
[The linker (link.exe) can](https://learn.microsoft.com/en-us/cpp/build/reference/stack-stack-allocations?view=msvc-170).